nemoguardrails/library/cleanlab/actions.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import logging
+import os
+from typing import Optional
+
+from nemoguardrails.actions import action
+from nemoguardrails.colang.v1_0.lang.utils import new_uuid
+
+log = logging.getLogger(__name__)
+
+
+@action(name="call cleanlab api", is_system_action=True)
+async def call_cleanlab_api(context: Optional[dict] = None):
```

Comment:
Nit: can be helpful to put in return type since you also have input types

Reply:
Added 3 return types.